{
  "parameters": {
    "api-version": "2022-05-01-preview",
    "failoverGroupName": "failover-group-test-3",
    "locationName": "Japan West",
    "resourceGroupName": "Default",
    "subscriptionId": "00000000-1111-2222-3333-444444444444"
  },
  "responses": {
    "200": {
      "body": {
        "name": "failover-group-test-3",
        "type": "Microsoft.Sql/locations/instanceFailoverGroups",
        "id": "/subscriptions/00000000-1111-2222-3333-444444444444/resourceGroups/Default/providers/Microsoft.Sql/locations/JapanWest/instanceFailoverGroups/failover-group-test-3",
        "properties": {
          "managedInstancePairs": [
            {
              "partnerManagedInstanceId": "/subscriptions/00000000-1111-2222-3333-444444444444/resourceGroups/Default/providers/Microsoft.Sql/managedInstances/failover-group-primary-mngdInstance",
              "primaryManagedInstanceId": "/subscriptions/00000000-1111-2222-3333-444444444444/resourceGroups/Default/providers/Microsoft.Sql/managedInstances/failover-group-secondary-mngdInstance"
            }
          ],
          "partnerRegions": [
            {
              "location": "Japan East",
              "replicationRole": "Secondary"
            }
          ],
          "readOnlyEndpoint": {
            "failoverPolicy": "Disabled"
          },
          "readWriteEndpoint": {
            "failoverPolicy": "Automatic",
            "failoverWithDataLossGracePeriodMinutes": 480
          },
          "replicationRole": "Primary",
          "replicationState": "CATCH_UP",
          "secondaryType": "Geo"
        }
      }
    },
    "202": {}
  },
  "operationId": "InstanceFailoverGroups_ForceFailoverAllowDataLoss",
  "title": "Forced failover of a failover group allowing data loss"
}